doc/source/conf.py

```diff
@@ -798,3 +799,40 @@ def setup(app):
     app.add_autodocumenter(AccessorMethodDocumenter)
     app.add_autodocumenter(AccessorCallableDocumenter)
     app.add_directive("autosummary", PandasAutosummary)
+
+
+####### This is an ignore list for broken links found in CI run checks for broken-linkcheck.yml
+
+linkcheck_ignore = [
```

Comment:
I see now where you got the `http://` and `https://` from: https://github.com/pandas-dev/pandas/actions/runs/6511956106/job/17688629505?pr=55246#step:6:9

https://github.com/pandas-dev/pandas/blob/a83f6aae255d7d88fe26ac12565c26253a95751b/pandas/io/formats/style_render.py#L998-L1001

These two need to be permanently ignored, since there is no broken link here, but rather an incomplete one.

```suggestion
linkcheck_ignore = [
    '^http://$',
    '^https://$',
```

- the `^` at the front means "The link has to start exactly here. No other character is allowed before"
- the `$` at the front means "The link has to stop exactly here. No other character is allowed after"

Have a look at the [documentation of the `re` module](https://docs.python.org/3/library/re.html). I'm also quite fond of https://regex101.com (don't forget to select Python in the left sidebar).